.github/workflows/test.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           PR_NUMBER: ${{ github.event.pull_request.number }}
           PR_BODY: "Test triggered by ${{ github.event.pull_request.html_url }}."
         run: |
-          ve1/bin/pytest tests/ --log-cli-level=WARNING --ignore=tests/functional/step_defs/test_submitted_charts.py --tb=short
+          ve1/bin/pytest tests/functional/step_defs/test_smoke_scenarios.py --log-cli-level=WARNING --tb=short
```

Comment:
It is here we need to run full test for the release process.

Also, a new requirement but we should only do smoke test to if PR is in development repo. If we are updating CI workflow in charts repo we should also run the full bucket.

Reply:
Addressed this new requirement as well.